### DISCUSSION TITLE
SSO from linux container

### DISCUSSION THREAD

**Discussion by haayman-imagem** [jlguenego/node-expose-sspi#95] [Q&A]
I don't know a thing about Kerberos, NTLM, etc. 

My application runs in a linux container in docker but I'd really want to use this library for SSO. It's so easy when it works, but to my disapointment I couldn't build it using a linux container

My question is: is this an impossible wish? 
What would be an alternative? Have a different container using windows to create a webtoken from the data obtained using req.sso? I wouldn't like this option since creating windows containers running node isn't officially supported.

Or should I mess around with kerberos config files? If so: I'm puzzled why it would work so perfectly config-less on windows and not on linux?

**Comment by jlguenego** [comment]
nothing is impossible, but I have no time to develop the equivalent library for linux.
When I decided to develop this library, it was to wrap some Win32 call. They do not work on Linux.
Sorry.
